core/server/update-check.js

```diff
@@ -38,6 +38,24 @@ var crypto   = require('crypto'),
     checkEndpoint = 'updates.ghost.org',
     currentVersion = config.ghostVersion;
 
+/**
+ * Attempts to parse string to JSON.
+ * @return {Object} is successful. Otherwise, undefined.
+ */
+function safeParseJson(json) {
```

Comment:
A convenience function like parse does make sense to be included in utils. But... this is a very naive implementation and ignores any errors. I think more is expected from a function in utils: 
a) accept a default value which will be returned in case of errors 
b) return where parsing failed
At the very least (a). It seems the parse behavior is dependent on caller because the only addition on top of `JSON.parse` is how to deal with errors. So I thought it was best to keep this local.